go/vt/vtgate/planbuilder/testdata/filter_cases.json

```diff
@@ -1150,7 +1150,7 @@
       "Original": "select Id from user where 1 in ('aa', 'bb')",
       "Instructions": {
         "OperatorType": "Route",
-        "Variant": "Scatter",
+        "Variant": "None",
```

Comment:
We are now using the `evalengine` to check if we can at plan-time evaluate expression. If we can, and if the result is `false`, we can use the `None` opcode which is very cheap.